farm.js

```diff
@@ -8,24 +8,30 @@ const feedCow = require('cow-foo/feed-cow');
 const feedBird = require('./lib/feed-bird');
 const eatLunch = require('./lib/eat-lunch');
 
+/** @todo: Fix feedCow to use Promises and throw away this wrapper */
```

Comment:
`feedCow` is an external library so lets assume it will never change.

How about throw this adapter into `lib/` and keep `main` clean.